docs/install/kubernetes.md

```diff
@@ -62,6 +57,23 @@ To learn about the various UI options and their usage, see:
 <a id="installing-the-chart"></a>
 ## Installing the chart
 
+### Chart Details
+
+The MLRun CE chart includes the following stack:
+
+* Nuclio - https://github.com/nuclio/nuclio
+* MLRun - https://github.com/mlrun/mlrun
+* Jupyter - https://github.com/jupyter/notebook (+MLRun integrated)
+* MPI Operator - https://github.com/kubeflow/mpi-operator
+* Minio - https://github.com/minio/minio/tree/master/helm/minio
```

Comment:
I would mention that the basic stack is called lite, If we will mention it in other places I think we should address its name here as well.